templates/form-template.php

```diff
@@ -25,6 +25,9 @@
 <form id="form_example" action="<?php echo esc_url( admin_url( 'admin-post.php' ) ); ?>" method="post">
 
 <?php
+	// Include form ID if available.
+	echo $form_id ? '<input type="hidden" name="formID" value="' . $form_id .'">' : '';
```

Comment:
`[phpcs] Concat operator must be surrounded by a single space (29,74)`

Reply:
I'll do an additional `phpcs` run in future PRs, thanks for noting that. As for this PR, there are other ones based on it, it's better to not create merge conflicts for such minor things.